File: core/src/main/java/com/pluxity/station/dto/StationUpdateInfo.java
package com.pluxity.station.dto;

import io.swagger.v3.oas.annotations.media.Schema;
import java.util.List;

public record StationUpdateInfo(
        @Schema(requiredMode = Schema.RequiredMode.REQUIRED) List<Long> lineIds,
        @Schema(requiredMode = Schema.RequiredMode.REQUIRED) List<String> stationCodes) {}

File: core/src/main/kotlin/com/pluxity/station/dto/StationUpdateInfo.kt
package com.pluxity.station.dto

import io.swagger.v3.oas.annotations.media.Schema

data class StationUpdateInfo(
    @field:Schema(requiredMode = Schema.RequiredMode.REQUIRED)
    val lineIds: List<Long>,
    @field:Schema(requiredMode = Schema.RequiredMode.REQUIRED)
    val stationCodes: List<String>,
)
